src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q10/PiperSaratoga.java

```diff
@@ -0,0 +1,56 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q10;
+
+/*
+A Piper Saratoga is a hobby plane with six seats that has a capacity to hold 1200 pounds on a flight.
+ Make a program that helps this plane's pilot determine how much luggage can be brought on the return flight from
```

Comment:
Also, the question asks how much luggage they can take back, not souvenirs, so I would just prompt the user for the weight of the gas and people, and then print out how heavy their luggage can be.